bot/cogs/random_cog/random_cog.py

```diff
@@ -2,6 +2,7 @@
 import random
 import asyncio
 import time
+import typing
```

Comment:
you can alias this to t if youd like `import typing as t` i find it makes for shorter code to read